# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit/MIROps.yaml
# Commit: 07f0b26b5449
# Full Hash: 07f0b26b54495095213f80a7808cc7ef90c7454d
# Author: Jonatan Klemets <jonatan.r.klemets@gmail.com>
# Date: 2021-12-02 09:42:49
# Regressor Bug: 1736060
# File Overlap Count: 1
# Description:
#   Bug 1736060 - Part 6: Implement import assertions for dynamic imports. r=mgaudet,arai
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D126046
# ==============================================================================

diff -r d9fa28df9787 -r 07f0b26b5449 js/src/jit/MIROps.yaml
--- a/js/src/jit/MIROps.yaml	Wed Dec 01 18:26:10 2021 +0000
+++ b/js/src/jit/MIROps.yaml	Wed Dec 01 18:26:10 2021 +0000
@@ -1001,6 +1001,7 @@
 - name: DynamicImport
   operands:
     specifier: Value
+    options: Value
   result_type: Object
 
 - name: Lambda